src/job_cache/eviction_policy.cpp

```diff
@@ -367,7 +367,8 @@ LRUEvictionPolicy::~LRUEvictionPolicy() {}
 int eviction_loop(const std::string& cache_dir, std::unique_ptr<EvictionPolicy> policy) {
   policy->init(cache_dir);
 
-  MessageParser msg_parser(STDIN_FILENO);
+  // Famous last words "a timeout of a year is plenty"
+  MessageParser msg_parser(STDIN_FILENO, 60 * 60 * 24 * 30 * 12);
```

Comment:
Yeah at a few levels 1) I checked the fuzz tests after running to ensure their size and that eviction was running at all and 2) We have the basic-lru test that just sort of smoke tests that its working at all and sets the sizes so that it can intentionally cause lru-evictions at specific points.

Reply:
I think we could add something to the fuzz testing to ensure 1) A certain hit rate occurs and 2) that the size never goes over what we expect it to

Reply:
That should be in a later PR I think however, we've tested it ok-ishly well here.